packages/components/src/accessible-popover/utils.js

```diff
@@ -7,7 +7,8 @@ import useResizeAware from 'react-resize-aware';
 /**
  * Internal dependencies
  */
-import { useIsomorphicLayoutEffect } from '../utils';
+// TODO: use the version from `@wordpress/compose`
+import { useIsomorphicLayoutEffect } from '../ui/utils';
```

Comment:
Currently not possible until `compose` is typed 😞 It's copied here for now for that reason.

Reply:
Composed is now typed! Rebasing on top of trunk will fix this :)